graph.js

```diff
@@ -183,6 +224,20 @@ app
 
     let logger = new Logger(0, { verbosity: getVerbosity(app) })
 
+    if (cmd.auth) {
+      keytar.getPassword('subgraph-hosting', cmd.node).then((authToken) => {
```

Comment:
Rework this logic to take both the stored access token and then one optionally passed in with `--access-token` into account.